app/src/main/res/values/strings.xml

```diff
@@ -188,37 +188,37 @@
     <string name="no_video">No video</string>
     <string name="audio">Audio</string>
     <string name="video">Video</string>
-    <string name="downloading">Downloading …</string>
+    <string name="downloading">Downloading…</string>
     <string name="player_autoplay">Autoplay</string>
     <string name="hideTrendingPage">Hide trending page</string>
     <string name="instance_frontend_url">URL to instance frontend</string>
     <string name="quality">Quality</string>
     <string name="behavior">Behavior</string>
-    <string name="player_summary">Quality and player behavior</string>
+    <string name="player_summary">Quality, and player behavior</string>
     <string name="seek_increment">Seek increment</string>
-    <string name="piped_summary">Piped is an open-source alternative web-frontend for YouTube provides the API we use. Without Piped, LibreTube wouldn\'t exist. Huge thanks to their devs!</string>
-    <string name="pauseOnScreenOff">Autopause</string>
-    <string name="pauseOnScreenOff_summary">Pause the player when the screen is turned off.</string>
-    <string name="autoplay_summary">Automatically play the next video when the current is finished.</string>
+    <string name="piped_summary">Piped is a libre alternative web-frontend for YouTube provides the API we use. Without it, LibreTube wouldn\'t exist. Huge thanks to their devs!</string>
+    <string name="pauseOnScreenOff">Auto-pause</string>
+    <string name="pauseOnScreenOff_summary">Pause playback when the screen is turned off.</string>
+    <string name="autoplay_summary">Auto-play the next video after the current one.</string>
     <string name="clonePlaylist">Clone playlist</string>
     <string name="reset">Restore defaults</string>
-    <string name="reset_message">Are you sure? This will log you out and reset all your settings!</string>
+    <string name="reset_message">Reset all settings and log out?</string>
     <string name="deleteAccount">Delete account</string>
     <string name="deleteAccount_summary">Delete your Piped account</string>
     <string name="account">Account</string>
     <string name="restore">Restore</string>
     <string name="watch_history">Watch History</string>
     <string name="watch_positions">Remember position</string>
-    <string name="watch_positions_summary">Restore last watch position</string>
-    <string name="auth_instance">Authentication instance</string>
-    <string name="auth_instance_summary">Use a different instance for authenticated calls.</string>
+    <string name="watch_positions_summary">Continue from last playback position</string>
+    <string name="auth_instance">Log in elsewhere</string>
```

Comment:
We should better leave that `Authentication instance`

Reply:
The benefit escapes me.
I think that may be the official term, but it seems the actual meaning alludes people not already technically adept.
Yes it is a setting for technical users, but it is also important that regular ones understand not to use it, and more importantly aren't alarmed by its presence.

The question that arises is whether an authentication instance is needed in its own right.
This might lead people to try to input something, not knowing that there is already a sane default.
If the user doesn't know what "authenticated calls" are, in all likelihood it could be that the user thinks
that is an extra benefit available when connecting to a "different instance" to do so.

The direct translations aren't any better, and the attempted ones are more misleading.
They are also longer.